reproducibility_project/src/engines/hoomd/project.py

```diff
@@ -83,7 +133,28 @@ def run_hoomd(job):
     integrator.methods = [nvt]
     sim.operations.integrator = integrator
     sim.state.thermalize_particle_momenta(filter=hoomd.filter.All(), kT=kT)
+
+    # Shrink step follows this example
+    # https://hoomd-blue.readthedocs.io/en/latest/tutorial/
+    # 01-Introducing-Molecular-Dynamics/03-Compressing-the-System.html
+    ramp = hoomd.variant.Ramp(A=0, B=1, t_start=sim.timestep, t_ramp=int(2e4))
+    initial_box = sim.state.box
+    L = job.sp.box_L_liq
+    final_box = hoomd.Box(Lx=L, Ly=L, Lz=L)
+    box_resize_trigger = hoomd.trigger.Periodic(10)
+    box_resize = hoomd.update.BoxResize(
+        box1=initial_box,
+        box2=final_box,
+        variant=ramp,
+        trigger=box_resize_trigger,
+    )
+    sim.operations.updaters.append(box_resize)
+    sim.run(2e4 + 1)
```

Comment:
Just for my own curiosity, what is the reasoning for 2e4 + 1?

Reply:
2e4 is the box shrink time, by running one more step the box is fully shrunk and the following assert line will evaluate to true

Reply:
Ah gotcha! I missed that.